microsoft-edge/toc.yml

```diff
@@ -958,47 +958,68 @@
 # /DevTools
 
 # =============================================================================
-# if top-of-page title is longer, add it to displayName comma-delimited list of lookup keywords
     - name: Microsoft Edge extensions
       items:
         - name: Overview of Microsoft Edge extensions
           href: extensions-chromium/index.md
 
+# -------------------------------------
         - name: What's new
           items:
             - name: Released features for Microsoft Edge Add-ons
               href: extensions-chromium/whats-new/released-features.md
+                  # add blogs
 
             - name: Roadmap for Microsoft Edge Add-ons
               href: extensions-chromium/whats-new/roadmap.md
 
-        - name: Getting started
+# -------------------------------------
+        - name: Support and feedback
```

Comment:
move "Support" TOC bucket to bottom of Extensions TOC bucket?

like in DevTools TOC bucket & in WebView2 TOC bucket

Rendered: 
https://review.learn.microsoft.com/microsoft-edge/extensions-chromium/publish/contact-extensions-team?branch=pr-en-us-2899

Reply:
moved Support to bottom